omnisette/src/remote_anisette_v3.rs

```diff
@@ -145,9 +146,15 @@ pub struct AnisetteData {
 }
 
 impl AnisetteData {
-    pub fn get_headers(&self) -> HashMap<String, String> {
-        // TODO time headers
+    pub fn get_headers(&self, serial: String) -> HashMap<String, String> {
+        let dt: DateTime<Utc> = Utc::now().round_subsecs(0);
+        println!("here {}", dt.format("%+").to_string());
+        
         HashMap::from_iter([
+            ("X-Apple-I-Client-Time".to_string(), dt.format("%+").to_string().replace("+00:00", "Z")),
+            ("X-Apple-I-SRL-NO".to_string(), serial),
+            ("X-Apple-I-TimeZone".to_string(), "UTC".to_string()),
+            ("X-Apple-Locale".to_string(), "en_US".to_string()),
```

Comment:
Maybe. I added it there because the anisette v1 returns those headers, and the v3 impl needs to return them too to maintain compatibility. Timestamps are also important for OTP verification, so I think they're good in the impl for the most part.

Reply:
Fair